src/plugins/intel_cpu/src/weights_cache.cpp

```diff
@@ -10,8 +10,6 @@
 namespace ov {
 namespace intel_cpu {
 
-const SimpleDataHash WeightsSharing::simpleCRC;
```

Comment:
Do we need to keep SimpleDataHash class anymore?

Reply:
Actually we don't. Removed, thanks